app/views/shortFormatPreSentenceReport/completed.scala.html

```diff
@@ -10,63 +10,71 @@
 
 @(reportForm: Form[ShortFormatPreSentenceReportData], encrypter: String => String, reviewPage: Int)
 
-    @implicitField = @{ FieldConstructor(govukElements.f) }
+@implicitField = @{ FieldConstructor(govukElements.f) }
 
-@main("Report saved - Short Format Pre Sentence Report", true, Some("completed")) {
+@main("Draft report saved - Short Format Pre Sentence Report", true, Some("cancelled")) {
```

Comment:
Looks like a cut and paste error here; I think this should be `Report saved` and `Some("completed")`